### PR TITLE
STUDIO-4266]  Hiding FileAlreadyExistsException

### PR DESCRIPTION
**Pull request by linhtnlK**
[STUDIO-4266](https://katalon.atlassian.net/browse/STUDIO-4266)

[STUDIO-4266]: https://katalon.atlassian.net/browse/STUDIO-4266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Test Listeners/KatalonReportListener.groovy [hunk 1]
```diff
@@ -20,7 +21,9 @@ public class KatalonReportListener {
                 try {
                     Files.copy(source, destination);
                 } catch (IOException e) {
-                    KeywordUtil.markWarning(ExceptionsUtil.getStackTraceForThrowable(e));
+                    if(!e instanceof FileAlreadyExistsException) {
```
Comment (Minh-Chien) [inline Test Listeners/KatalonReportListener.groovy:24]: Why don't we use `!(e instanceof FileAlreadyExistsException)` ?

**Review by linhtnlK — COMMENTED** [review]
Update condition

**Review by Minh-Chien — COMMENTED** [review]
Have you verified this solution =)) on your machine?

**Comment by linhtnlK** [comment]
> ied this solution =)) on your machine?

I tested a. Tks 
image

**Review by Minh-Chien — APPROVED** [review]
👍

**Comment by Minh-Chien** [comment]
Please help to create a PR on this repo: https://github.com/katalon-studio/katalon/ after this PR merged.

**Pull request merged**